### PR TITLE
(WIP) Docs: Update readme

### PR DESCRIPTION
**Pull request by KSerrania**
- Add description of the project
- Update app setup to reflect current state of the project
- Add contribution guidelines

@oxlay @Gandem could you help me with wording / rephrasing for better clarity?

### CODE REVIEW

**Comment by Gandem** [comment]
I'll add my suggestions in a review so we can further discuss the wording !

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # bonjour-reflector
 
+## About this project
+
+This projects aims to make bonjour devices (such as printers, chromecasts, ...) discoverable and usable by other devices located in different VLANs.
```
Comment (Gandem) [inline README.md]: I'd add something about how this is similar to avahi-reflector, but in addition permits a fine-grained control of how Bonjour traffic is reflected between VLANs.

Comment (Gandem) [reply README.md]: And this project* typo

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+This is done by intercepting mDNS packets and forwarding them to the correct VLANs (for the request packets, the VLANs where the bonjour devices are located; for the response packets, the VLANs where the searching devices are located).
```
Comment (Gandem) [inline README.md]: Let's split this in bullet points to make it clearer maybe ?

Comment (Gandem) [reply README.md]: Should we also add a sentence about how this interception is done ? (the bonjour-reflector should have on interface with all the concerned vlans on **tagged** for example)

**Comment by oxlay** [comment]
I reorganized a few bits in the README. Feel free to discuss these changes if you think the README could be improved :)

**Pull request merged**